### ISSUE TITLE
How to install the framework

### ISSUE THREAD

**Issue by breiko83** [mirek/YAML.framework#18]
Hello, I need the YAML Framework in my iOS project. Can you help me?
I have tried to drag the .framework folder but headers are missing. Which are the steps to follow?

**Comment by jimbojsb** [comment]
did you ever figure this out @breiko83

**Comment by breiko83** [comment]
I think I did but this is so old that I don't remember. Have you got the same issue? Otherwise I should close this.

**Comment by jimbojsb** [comment]
Yeah exact same problem. Tried To install with cocoa pods but that failed as well.

**Comment by neuralmer** [comment]
I had some trouble installing as well, but I added something like the following to may Podfile:
```
use_frameworks!
target 'some-build-target' do`
  pod 'YAML-Framework'
end
```
... and refreshed the pods.

Added an include to my sources:
`#import <YAML-Framework/YAMLSerialization.h>`

At that point, Xcode couldn't find the header, and it appears that something is either wrong with my project (likely), or maybe something with the configurations for the YAML-Framework that is preventing the header search path from including Pods/Headers/Private or Pods/Headers/Public.  I can see the header is there under YAML-Framework, but looking at the build log, I could see this path wasn't included.  As a short-term fix, I added to the paths listed in HEADER_SEARCH_PATHS: `${PODS_ROOT}/Headers/Private`

This added the correct -I argument on the CompileC command-line.  I see PODS_ROOT being defined in Pods.debug.xcconfig and similar, so I am guessing this has to be included in the configuration for the target.
